po/fr.po

```diff
@@ -0,0 +1,486 @@
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: vaults\n"
+"POT-Creation-Date: 2023-02-26 20:16+0100\n"
+"PO-Revision-Date: 2023-02-26 19:12+0100\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.1.1\n"
+"X-Poedit-Basepath: ..\n"
+"X-Poedit-KeywordsList: gettext\n"
+"X-Poedit-SearchPath-0: src\n"
+"X-Poedit-SearchPath-1: data\n"
+
+#: src/backend/cryfs.rs:128 src/backend/cryfs.rs:145
+#: src/backend/gocryptfs.rs:149
+msgid "An unknown error occurred."
+msgstr "Une erreur inconnue s'est produite"
+
+#: src/backend/cryfs.rs:129
+msgid "Invalid arguments were given."
+msgstr "Des arguments invalides ont été donnés"
+
+#: src/backend/cryfs.rs:130 src/backend/gocryptfs.rs:135
+msgid "The password is wrong."
+msgstr "Mot de passe erroné"
+
+#: src/backend/cryfs.rs:131 src/backend/gocryptfs.rs:138
+msgid "The password is empty."
+msgstr "Le mot de passe est vide"
+
+#: src/backend/cryfs.rs:132
+msgid "The format of the encrypted data directory is too new for this CryFS version. Please update CryFS."
+msgstr "Le format du dossier chiffré est trop récent pour cette version de CryFS. Veuillez mettre à jour CryFS"
```

Comment:
The sentences are missing a period at the end.

Reply:
Thanks I added these missing periods.